src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -171,7 +171,7 @@ public override int Run(AnalyzeOptions analyzeOptions)
         {
             Stopwatch stopwatch = null;
 
-            if (analyzeOptions.Traces.Where(s => s == "ScanTime").Any())
+            if (analyzeOptions.Trace.Where(s => s == "ScanTime").Any())
```

Comment:
You should be using the Traces property in the base class.

[https://github.com/microsoft/sarif-sdk/blob/140d730afd838e7e168985bfda38edd4ead7d297/src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs#L86](https://github.com/microsoft/sarif-sdk/blob/140d730afd838e7e168985bfda38edd4ead7d297/src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs#L86)

Reply:
Please see other comments first, here can either use the current one or base one since current one incudes all.
The base one will throw null exception where 
`this.Trace` is null, so I kept the current one I have.

``` CSharp
public DefaultTraces Traces
        {
            get
            {
                defaultTraces ??=
                    this.Trace.Any()
                        ? (DefaultTraces)Enum.Parse(typeof(DefaultTraces), string.Join(",", this.Trace))
                        : DefaultTraces.None;

                return this.defaultTraces.Value;
            }
        }
```